code/modules/mob/new_player/new_player.dm

```diff
@@ -347,6 +347,11 @@
 					return
 				to_chat(src, "<span class='notice'>Vote successful.</span>")
 
+/datum/game_mode/proc/observe_respawn_message()
+	return "\nYou might have to wait a certain time to respawn or be unable to, depending on the game mode!"
+
+/datum/game_mode/distress/observe_respawn_message()
```

Comment:
This will conflict with the job refactor, but as soon as it's merged you can turn this proc into `/datum/game_mode/infestation/observe_respawn_message()` and it will work for both Crash and Distress.